frontend/src/lib/utils/neuron.utils.ts

```diff
@@ -947,3 +947,60 @@ export const maturityLastDistribution = ({
 
 export const neuronDashboardUrl = ({ neuronId }: NeuronInfo): string =>
   `https://dashboard.internetcomputer.org/neuron/${neuronId.toString()}`;
+
+export const getTopicTitle = ({
+  topic,
+  i18n,
+}: {
+  topic: Topic;
+  i18n: I18n;
+}): string => {
+  const mapper = {
```

Comment:
What do you think about adding a types to not forget about mapper when we add a new Topic?
```suggestion
  const mapper: Record<Topic, string> = {
```

Same to the `getTopicSubtitle`.

Reply:
There is a test to make sure no topics are missed, but adding a type seems like a good idea anyway.
Thanks. Done.